### DISCUSSION TITLE
Ha integration

### DISCUSSION THREAD

**Discussion by Welsyntoffie** [blakeblackshear/frigate#12116] [General]
Since frigate 14 I have been seeing a raise in the integration reporting unavailable at random times.
I have an automation that sets Night and Day settings whenever the status changes to "running" and also an automation with telegram. Been getting a lot of random times that frigate is running. Seems to go unavailable for about 40 seconds at a time.

**Comment by ofcedwards** [comment]
+1 on this. I haven't seen any consistency to it to track down the issue, but I've intermittently had the HA frigate integration report that frigate is unavailable despite frigate being up. I have HA set to send notifications when the integration reports running/unavailable which is the only way I've noticed this.

**Comment by Welsyntoffie** [comment]
I like my notifications to be accurate. So I had to create a helper toggle. Then check for frigate unavailable for 50 seconds then switch off the helper. Similar for the on. I use this now to send the notifications. The intermittent unavailable state lasts about 45 seconds.

**Comment by blakeblackshear** [comment]
Could this be related? https://github.com/blakeblackshear/frigate-hass-integration/issues/686#issuecomment-2184260805

**Reply by ofcedwards** [reply]
For me, it's possible. I have the same setup as what's described there. I've only had the integration incorrectly switch to unavailable  maybe 3 times in the last few weeks though. Not enough to really even try to figure out the cause. Sounds like @Welsyntoffie and the people in that thread have much more frequent issues.

**Comment by Welsyntoffie** [comment]
Does not look related, I am not using Mail and Packages addon/integrations.
I have re-pulled all container images.  Cleaned up some I dont need anymore.

Frigate docker logs I found this...
`2024-06-23 11:05:43.661016991  ::ffff:127.0.0.1 - - [23/Jun/2024:11:05:43 +0200] "\x16\x03\x01\x01\x16\x01\x00\x01\x12\x03\x03Rr\x9FA\xE4\x1C\xE2X<\xACd[%O\xF5\x19\xB3\xB2\xC9\x1ES\xF1\x84\xBB\x95\xCA\x5C\xF4Z\x00~Z c\xDDr\xAF[7*\xE3\xFD\xA3#;Z\xED\xDD\xF2\xED\x02ON\x81\xD8\xA4\xA8\xB5\x975(\x1EP\xFF\xA4\x00>\x13\x02\x13\x03\x13\x01\xC0,\xC00\x00\x9F\xCC\xA9\xCC\xA8\xCC\xAA\xC0+\xC0/\x00\x9E\xC0$\xC0(\x00k\xC0#\xC0'\x00g\xC0" 400 157 "-" "-" "-"
2024-06-23 11:05:43.661862242  [ERROR] Missing fingerprint from current nginx TLS cert`

Edit: 12:15
  Disabled TLS in config. Solved the fingerprint error message.

**Comment by Welsyntoffie** [comment]
Running status resolved as per [this](https://github.com/blakeblackshear/frigate/discussions/12217) post.
Some cameras has corrupted data, causing ffmpeg to restart and frigate to hang for awhile.
All barrel power jacks removed and wires soldered directly.
